server/catgenome/src/main/java/com/epam/catgenome/manager/reference/ReferenceSecurityService.java

```diff
@@ -163,4 +163,11 @@ public Reference updateSpecies(Long referenceId, String speciesVersion) {
     public MotifSearchResult getTableByMotif(final MotifSearchRequest motifSearchRequest) {
         return motifSearchManager.search(motifSearchRequest);
     }
+
+    @PreAuthorize(ROLE_USER)
+    public StrandedSequence getNextOrPrevMotif(final MotifSearchRequest motifSearchRequest, final boolean forward) {
```

Comment:
We have two methods in controller level, one at security level and then two again at manager level? I think it makes sense to have two methods at all levels.